spec/models/lesson_preview_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe LessonPreview, type: :model do
+  it { is_expected.to validate_presence_of(:content) }
```

Comment:
Lets add a test for the max length validation too:
```suggestion
  it { is_expected.to validate_presence_of(:content) }
  it { is_expected.to validate_length_of(:content).is_at_most(70_000) }
```